spring-cloud-kubernetes-test-support/src/main/resources/current-images.txt

```diff
@@ -0,0 +1,7 @@
+busybox:1.35
```

Comment:
I would recommend unit testing this somehow. Possibly you can make a test that dumps the images from k3s ctl and fails if it has something that isn't in here or visa versa? Just a thought, as this is very likely to drift.

Reply:
another way could be to make this like DockerImages.go, and give it a main function that prints out a list like this. Then, as long as you checked out the source first and installed go, you could run that main and also have no drift.

Reply:
p.s. I meant DockerImages.java ;) ok you can also solve this by making a comment in the java code that defines the istio versions saying "// please update current-images.txt when you update this"

because at the end of the day, it is about drift making the infrastructure added here serve no value.

Reply:
👌